packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -71,13 +71,14 @@
-  'arrow':'<svg viewBox="0 0 10 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.29289 0.292893C4.68342 -0.0976311 5.31658 -0.0976311 5.70711 0.292893L9.70711 4.29289C10.0976 4.68342 10.0976 5.31658 9.70711 5.70711C9.31658 6.09763 8.68342 6.09763 8.29289 5.70711L6 3.41421V11C6 11.5523 5.55228 12 5 12C4.44772 12 4 11.5523 4 11V3.41421L1.70711 5.70711C1.31658 6.09763 0.683417 6.09763 0.292893 5.70711C-0.0976311 5.31658 -0.0976311 4.68342 0.292893 4.29289L4.29289 0.292893Z" fill="%%COLOR%%"/></svg>',
   'dash':'<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 10 2"><path fill="%%COLOR%%" d="M0 0h10v2H0z"/></svg>',
+  'double-arrow':'<svg viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M2.29289 0.292893C2.68342 -0.0976311 3.31658 -0.0976311 3.70711 0.292893L5.70711 2.29289C6.09763 2.68342 6.09763 3.31658 5.70711 3.70711C5.31658 4.09763 4.68342 4.09763 4.29289 3.70711L4 3.41421V10C4 10.5523 3.55228 11 3 11C2.44772 11 2 10.5523 2 10V3.41421L1.70711 3.70711C1.31658 4.09763 0.683417 4.09763 0.292893 3.70711C-0.0976311 3.31658 -0.0976311 2.68342 0.292893 2.29289L2.29289 0.292893Z" fill="%%COLOR%%"/><path fill-rule="evenodd" clip-rule="evenodd" d="M9 12C8.73478 12 8.48043 11.8946 8.29289 11.7071L6.29289 9.70711C5.90237 9.31658 5.90237 8.68342 6.29289 8.29289C6.68342 7.90237 7.31658 7.90237 7.70711 8.29289L8 8.58579L8 2C8 1.44771 8.44772 1 9 1C9.55229 1 10 1.44771 10 2L10 8.58579L10.2929 8.29289C10.6834 7.90237 11.3166 7.90237 11.7071 8.29289C12.0976 8.68342 12.0976 9.31658 11.7071 9.70711L9.70711 11.7071C9.51957 11.8946 9.26522 12 9 12Z" fill="%%COLOR%%"/></svg>',
+  'external':'<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 12 12"><path fill="%%COLOR%%" fill-rule="evenodd" d="M10.462 1.809A.499.499 0 0010 1.5H6.5a.5.5 0 000 1h2.293L4.646 6.646a.5.5 0 10.708.708L9.5 3.207V5.5a.5.5 0 001 0V2v-.003a.5.5 0 00-.038-.188zM3.067 2.5a.567.567 0 00-.567.567v5.866a.567.567 0 00.567.567h5.866a.567.567 0 00.567-.567V8a.5.5 0 011 0v.933A1.567 1.567 0 018.933 10.5H3.067A1.567 1.567 0 011.5 8.933V3.067A1.567 1.567 0 013.067 1.5H4a.5.5 0 010 1h-.933z" clip-rule="evenodd"/></svg>',
+  'search':'<svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon"><path fill-rule="evenodd" clip-rule="evenodd" d="M8.75739 10.1716C7.96696 10.6951 7.01908 11 6 11C3.23858 11 1 8.76142 1 6C1 3.23858 3.23858 1 6 1C8.76142 1 11 3.23858 11 6C11 7.01908 10.6951 7.96696 10.1716 8.75739L12.6908 11.2766C13.0813 11.6671 13.0813 12.3003 12.6908 12.6908C12.3003 13.0813 11.6671 13.0813 11.2766 12.6908L8.75739 10.1716ZM9 6C9 7.65685 7.65685 9 6 9C4.34315 9 3 7.65685 3 6C3 4.34315 4.34315 3 6 3C7.65685 3 9 4.34315 9 6Z" fill="%%COLOR%%"/></svg>'
```

Comment:
We differentiate between Icons and UI Indicators within Odyssey; this implementation is part of that distinction.

Icons are content - whether semantic or decorative - that can be swapped out, restyled, etc.

UI Indicators (like a Select caret or Checkbox check) are served as background images because they should not be modified or replaced. 

There are a few cases where we cannot easily use SVGs for UI Indicators (e.g. pseudo-elements), so we've decided to maintain the same implementation for all Icons (SVG) and UI Indicators (background).

In both cases, they start as SVGs, but the UI Indicators don't have the same requirements our Icons do.

I've created a ticket to help us make this more clear inside the code: https://github.com/okta/odyssey/issues/701

Reply:
For the same reasons, the `Copy` icon should be here then, right? Wondering where we draw the line. Or maybe I'm reading it wrong.

Reply:
It depends on how you think about "Copy" 😈 

If it's a Button that you click, then the Copy Icon is content for the Button, replacing the text "Copy".

If there were a special Copy Button, only used for copying things and always using the symbol, then we'd have a UI Indicator for it.

(We're stilling trying to find the best ways of delineating between these, even on the Design side, so thanks for bearing with us.)

Reply:
Fair enough! I was anyways nitpicking at this point. Good for now!